README.md

```diff
@@ -37,6 +37,34 @@ curl -s https://get.helm.sh | bash
   - the CI build pipeline: [ ![Download](https://api.bintray.com/packages/deis/helm-ci/helm-classic/images/download.svg) ](https://bintray.com/deis/helm-ci/helm-classic/_latestVersion#files)
 2. Unzip the package and make sure `helmc` is available on the PATH.
 
+### Migration Notes
+
+If you are a user of the original Helm tool (versions prior to v0.7.0), take note that Helm Classic is a _re-branding_ of that tool-- new name, same great taste!
+
+__Helm Classic is fully compatible with existing Helm charts!__
+
+Anyone upgrading to Helm Classic from an older version should complete the following steps to fully replace their existing tool with Helm Classic. Doing so will clear the path for eventual installation of the new and improved Helm ([kubernetes/helm][k8s-helm]).
```

Comment:
"upgrading to Helm Classic" sounds a little weird - maybe "switching to Helm Classic"?